src/NServiceBus.AcceptanceTests/Sagas/When_adding_state_to_context.cs

```diff
@@ -39,8 +39,7 @@ public SagaEndpoint()
             {
                 EndpointSetup<DefaultServer>(c =>
                 {
-                    c.EnableFeature<TimeoutManager>();
-                    c.Pipeline.Register<BehaviorWhichAddsThingsToTheContext.Registration>();
+                    c.Pipeline.Register(new BehaviorWhichAddsThingsToTheContext(), "adds some data to the context");
```

Comment:
You can add the context data using SendOptions instead of a behavior? (less code?)